application/src/test/scala/com/azavea/franklin/api/services/FiltersFor.scala

```diff
@@ -0,0 +1,191 @@
+package com.azavea.franklin.api.services
+
+import cats.Semigroup
+import cats.data.NonEmptyList
+import cats.implicits._
+import com.azavea.franklin.database.SearchFilters
+import com.azavea.stac4s.{StacCollection, StacItem, TemporalExtent, TwoDimBbox}
+import geotrellis.vector.Extent
+import io.circe.optics._
+import io.circe.syntax._
+
+import java.time.Instant
+
+object FiltersFor {
+
+  val datetimePrism = JsonPath.root.datetime.as[Instant]
```

Comment:
This prism will always fail given current generators, but an upgrade to stac4s occasionally to generate items with datetimes should automatically enhance tests that rely on the datetime field to exist (thanks, @scala-steward!)